website/components/News.tsx

```diff
@@ -10,8 +10,8 @@ import { mq } from 'utils/mediaQuery';
 const newsContainer = css`
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  padding-bottom: ${spacing[4]}px;
+  justify-content: top;
+  padding-top: ${spacing[2] - 1}px;
```

Comment:
if we're going to make this a calculation, can we explain where the 1px is coming frrom? otherwise we can just do 7px

Reply:
Optical alignment where the spacing is still based on our spacing tokens. I'll comment.

Reply:
Thank you!